tests/config-import.js

```diff
@@ -0,0 +1,173 @@
+import { test } from 'brittle'
+import { readConfig } from '../src/config-import.js'
+
+test('config import - loading', async (t) => {
+  t.exception(async () => await readConfig(''), /ENOENT/, 'file not found')
```

Comment:
[Brittle's docs][0] say that "if a `t.exception` is async, then you're supposed to await it." I think we need to add `await` before each of these calls.

[0]: https://www.npmjs.com/package/brittle#texceptionpromisefunctionasync-function-error-message